doc/api/source_map.md

```diff
@@ -0,0 +1,85 @@
+# Source Map
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!--name=source_map-->
+
+Exposes an API for interacting with Node.js' internal cache of source maps.
+
+To enable source map parsing, Node.js must be run with the flag
+[`--enable-source-maps`][], or with code coverage enabled by setting
+[`NODE_V8_COVERAGE=dir`][].
+
+```js
+const source_map = require('source_map');
+```
+
+The APIs and implementation are subject to change at any time.
+
+## `source_map.findSourceMap(path[, error])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string}
+* `error` {Error}
+* Returns: {source_map.SourceMap}
+
+`path` should be the resolved path for the file that you wish to fetch a
+corresponding source map for.
+
+`error` if you are interacting with source maps in an exceptional flow, e.g.,
+having overridden [`Error.prepareStackTrace(error, trace)`][], you should pass
+the error instance as a second parameter to `findSourceMap`.
```

Comment:
What's the expected behavior here when `--enable-source-maps` is *not* enabled.

Reply:
If `--enable-source-maps` or `NODE_V8_COVERAGE` have not been set, then any call to `findSourceMap` will fail to find a source map corresponding to the path, and `null` will be the return value.